yt_dlp/extractor/amcnetworks.py

```diff
@@ -65,15 +65,39 @@ class AMCNetworksIE(ThePlatformIE):
     def _real_extract(self, url):
         site, display_id = re.match(self._VALID_URL, url).groups()
         requestor_id = self._REQUESTOR_ID_MAP[site]
-        properties = self._download_json(
-            'https://content-delivery-gw.svc.ds.amcn.com/api/v2/content/amcn/%s/url/%s' % (requestor_id.lower(), display_id),
-            display_id)['data']['properties']
+        page_data = self._download_json(
+            'https://content-delivery-gw.svc.ds.amcn.com/api/v2/content/amcn/%s/url/%s'
+            % (requestor_id.lower(), display_id), display_id)['data']
+        properties = page_data.get('properties')
```

Comment:
```suggestion
        properties = page_data.get('properties') or {}
```

Reply:
Better yet, page_data['properties'] will result in an extractor error if the key is missing.

Reply:
Since the basic functionality of the extractor could work even without the properties key, it is best to not throw error

Reply:
Huh? If the properties key is missing, the extractor can't find a video.

Reply:
how so? `properties` is only used to get `video_category` and thumbnails, no? The actual video is found using `page_data['children'][n]['properties']`

Reply:
> The actual video is found using `page_data['children'][n]['properties']`

`properties` is right there in your text. If that key's missing, no video will be found.

Reply:
`page_data['properties']` can be missing if `page_data['children'][n]['properties']` exists!

Reply:
Well, it can't be missing if `page_data['properties']['videoPid']` is ever needed but that can cause the extractor error instead.